src/views/enter-registered-office-address.njk

```diff
@@ -11,6 +11,7 @@
   {% set address = {} %}
 {% endif %}
 
+{% set pageTitle = i18n.address.chooseAddress.registeredOfficeAddress.title %}
```

Comment:
you have duplicate pageTitle variables being set here, the chooseAddress one looks like it is not needed here